detox/test/android/app/build.gradle

```diff
@@ -85,7 +80,9 @@ dependencies {
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
     implementation project(path: ':@react-native-community_slider')
 
-    fromSourceImplementation(project(path: ':ReactAndroid'))
+    if (gradle.ext.rnInfoLT.isRn68OrLower) {
+        fromSourceImplementation(project(path: ':ReactAndroid'))
```

Comment:
So this evidently eliminates our ability to build from source in RN 69! I'm not sure we're okay with that TBH.